lib/rucio/core/rse.py

```diff
@@ -40,9 +39,8 @@
 from rucio.db.sqla.constants import (RSEType, ReplicaState)
 from rucio.db.sqla.session import read_session, transactional_session, stream_session
 
-if TYPE_CHECKING:
-    from typing import Any, Dict, Optional, Sequence
-    from sqlalchemy.orm import Session
+from typing import Any, Dict, Optional, Sequence, Union
+from sqlalchemy.orm import Session
```

Comment:
These imports aren't sorted correctly. `typing` is a standard module and `sqlalchemy.orm` is already imported earlier.